src/Popover/Popover.js

```diff
@@ -366,6 +366,11 @@ Popover.propTypes = {
    * Useful to extend the style applied to components.
    */
   classes: PropTypes.object.isRequired,
+  /**
+   * A node, component instance, or function that returns either.
+   * The `container` will passed to the Modal component.
+   */
+  container: PropTypes.oneOfType([PropTypes.object, PropTypes.func]),
```

Comment:
@oliviertassinari Just wanted to verify if this is the correct way to fix the no validation linting error message when we are intercepting props being passed through to child components.

Reply:
I noticed it gets added to the docs API page which doesn't seem right :(

Reply:
Looks good 👍 . Run `yarn docs:api` to update the docs.